topology/IFAE/PIC/PIC-CachingInfrastructure.yaml

```diff
@@ -0,0 +1,25 @@
+GroupDescription: PIC caching infrastructure
+GroupID: 532
+Production: False
+Resources:
+  PIC_STASHCACHE_CACHE:
+    Active: true
+    ContactLists:
+      Administrative Contact:
+        Primary:
+          ID: bcef3fbf616e1a28248f5ad1920ec067151a44dc
+          Name: Carles Acosta
+      Security Contact:
+        Primary:
+          ID: bcef3fbf616e1a28248f5ad1920ec067151a44dc
+          Name: Carles Acosta
+    Description: StashCache cache server for PIC
+    FQDN: xcachevirgo.pic.es
+    ID: <ID>
```

Comment:
@Aco7 I missed this resource Id can you use:

1048

Reply:
Done. Thanks